wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/en.md

```diff
@@ -14,24 +14,35 @@ Modders aiming to apply to the Beatmap Nominators must fulfil the following crit
 - They have accumulated at least 150 or 200 [kudosu](/wiki/Modding/Kudosu) in total, depending on the mode they will apply to.
 - They provide 2 to 4 of their mods which they think qualify them to be a part of the Beatmap Nominators.
 
-### What qualifies as a mod post
-
-- Upon signing up, the website will automatically calculate your kudosu score from the last 3 months.
-- A qualified mod post is a user's posts on a beatmap discussion, in which at least one suggestion/problem got awarded with an upvote, and has been created within the 3 months.
-- The game mode that you modded beatmaps in does not matter as your activity in the modding community as a whole is being measured and not the activity in a certain game mode.
-- The point in time at which the kudosu has been awarded to your mod post **does** matter, as the calculator embed in the website will fetch the data from your own Modding History events.
-
 ### Modding expectations
 
 Submitted mods should show that a user is capable of analysing and improving maps to a standard fit for the Beatmap Nominators.
 
 The following are modding traits expected of Beatmap Nominators. Failure to display any of these traits may result in a rejected application.
 
-- Coverage of all difficulty levels
-- Identification of unrankable issues, including ones tools can't detect, such as incorrect timing or metadata
-- Comparison between parts of a map to support issues or suggestions
-- Commentary about a wide variety of map elements, such as rhythm, spacing, movement, intensity, contrast, and consistency
-- Identification of both isolated issues and general map-wide issues
+- **Coverage of all difficulty levels**
+- **Identification of unrankable issues, including ones tools can't detect, such as incorrect timing or metadata**
+- **Comparison between parts of a map to support issues or suggestions**
+- **Commentary about a wide variety of map elements, such as rhythm, spacing, movement, intensity, contrast, and consistency**
+- **Identification of both isolated issues and general map-wide issues** 
```

Comment:
should also add `Consideration of mappers' intentions when identifying issues and giving suggestions` after this to align with expectations written elsewhere